frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx

```diff
@@ -256,6 +291,18 @@ function OutputForDocModal({
           </Typography.Text>
         </div>
         <div className="output-doc-gap" />
+        <div className="lmm-profile-outputs">
+          <Tabs defaultActiveKey="0" onChange={handleTabChange}>
+            <TabPane tab={<span>{"Default"}</span>} key={"0"}></TabPane>
+            {adapterData?.map((adapter, index) => (
+              <TabPane
+                tab={<span>{adapter?.llm_model}</span>}
+                key={(index + 1)?.toString()}
+              ></TabPane>
+            ))}
+          </Tabs>{" "}
```

Comment:
@jagadeeswaran-zipstack Looks like the same block of code is reused in JsonView.jsx component. Can we move this to a separate component if possible?